tests/unit/reco_utils/evaluation/test_python_evaluation.py

```diff
@@ -35,7 +35,26 @@
 def rating_true():
     return pd.DataFrame(
         {
-            DEFAULT_USER_COL: [1, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 1, 1,],
+            DEFAULT_USER_COL: [
+                1,
```

Comment:
Is this reformatting due to black? It looks less readable, doesn't it? How about #fmt off / on around this function?

Reply:
yeah black did that, i can set it back